spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,12 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  it 'can return a list of up to 10 quotes based on search parameters' do
```

Comment:
Did you mean to have this test in your `quotes/index_spec.rb` :) 
The only thing that I would add is
```
expect(page).to have_content("Quote: You can't shake hands with a clenched fist")
```
The `Quote` part since it is also on the page. Particular I know lol